pipeline-steps/deploy-service.yaml

```diff
@@ -10,6 +10,7 @@ parameters:
   setClusterNumbersEnvVar: false
   action: ''
   terraformSubscriptionID: ''
+  backendAzureRmResourceGroupName: ''
 
 steps:
   - template: steps/set-build-repo-suffix-env-var.yaml@cnp-azuredevops-libraries
```

Comment:
I dont think this step is needed, the first step inside the `steps/terraform.yaml@cnp-azuredevops-libraries` is doing it as well.
You didnt make this change but worth checking if it can be removed.

Reply:
removed as duplicated under steps/terraform.yaml@cnp-azuredevops-libraries like this

  - template: ./set-build-repo-suffix-env-var.yaml